src/users/dtos/update-user.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { IsEmail, IsOptional, IsString, IsUrl, MaxLength, MinLength } from 'class-validator';
+
+export class UpdateUserDto {
+	@IsOptional()
+	@IsEmail({}, { message: 'Please provide a valid email address' })
+	email?: string;
+
+	@IsOptional()
+	@IsString()
+	@MinLength(3, { message: 'Username must be at least 3 characters long' })
```

Comment:
```suggestion
	@MinLength(<constant>, { message: 'Username must be at least 3 characters long' })
```

những giá trị number literal, ví dụ như 3, 20, 6... thì em khai báo constant, khi cần sử dụng thì gọi ra nhé.
tương tự như vậy đối với các literal khác, ví dụ `string`.
làm như vậy để tránh vấn đề hardcode